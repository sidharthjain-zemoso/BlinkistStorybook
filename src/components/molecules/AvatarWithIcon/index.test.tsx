import { render, screen } from "@testing-library/react";
import AvataarWithIcon from ".";

test("renders avatar with icon component", () => {
  render(<AvataarWithIcon>A</AvataarWithIcon>);

  const avatarElement = screen.getByTestId("account-avatar");
  const textElement = screen.getByText("A");
  expect(textElement).toBeDefined();
  expect(avatarElement).toMatchSnapshot();
});
