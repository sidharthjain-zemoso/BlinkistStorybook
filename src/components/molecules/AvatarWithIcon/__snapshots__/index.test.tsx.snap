// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders avatar with icon component 1`] = `
<div
  class="undefined makeStyles-root-1"
  data-testid="account-avatar"
  style="align-items: center;"
>
  <div
    class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-1cr0a2k-MuiAvatar-root"
  >
    <h6
      class="MuiTypography-root MuiTypography-subtitle2 css-ati914-MuiTypography-root"
      style="text-align: center;"
    >
      A
    </h6>
  </div>
  <img
    alt="icon"
    src="/res/icons/down-arrow.png"
    style="margin-right: 5px; height: 24px; width: 24px;"
  />
</div>
`;
